pkg/opensearch/response_parser_test.go

```diff
@@ -2668,14 +2668,21 @@ func TestProcessTraceSpans_creates_correct_data_frame_fields(t *testing.T) {
 
 	// logs
 	sortedLogs := sortLogsByTimestamp(series.Fields[11], t)
-	assert.Equal(t, sortedLogs[0].Timestamp, 1697615918486)
 	require.Equal(t, 2, len(sortedLogs))
-
-}
-
-type KeyValue struct {
-	Key   string
-	Value any
+	assert.Equal(t, sortedLogs[0].Timestamp, int64(1697615918486))
+	assert.Equal(t, sortedLogs[0].Name, "redis timeout")
+	// assert log attribute values
+	for i, logField := range sortedLogs[0].Fields {
```

Comment:
are we using the key from sortedLogs[0] and the value from sortedLogs[1] for a reason? It's not necessarily wrong (i assume they're the same keys?) but it just seems kind of odd

Reply:
Definitely an oversight, fixed